core-bundle/contao/languages/en/default.xlf

```diff
@@ -2526,14 +2526,20 @@
         <source>New</source>
       </trans-unit>
       <trans-unit id="DCA.pastenewtop.1">
-        <source>Create a new element after element ID %s</source>
+        <source>Create a new element at the top</source>
       </trans-unit>
       <trans-unit id="DCA.pastenewafter.0">
         <source>New</source>
       </trans-unit>
       <trans-unit id="DCA.pastenewafter.1">
         <source>Create a new element after element ID %s</source>
       </trans-unit>
+      <trans-unit id="DCA.pastenewinto.0">
+        <source>New</source>
```

Comment:
Not _New into_ and _New after_ in line 2532?

Reply:
changed in 2db2131fdbc4aa8e0bab264ab814f713532b37ae